zerver/openapi/zulip.yaml

```diff
@@ -4533,6 +4533,19 @@ paths:
                                             In Zulip 7.0 (feature level 159), `Nobody` was added as an option to
                                             `edit_topic_policy` enum.
                                         - $ref: "#/components/schemas/GroupSettingValue"
+                                    can_resolve_topics_group:
+                                      allOf:
+                                        - description: |
+                                            A [group-setting value](/api/group-setting-values) defining
+                                            the set of users who have permission to resolve topics
+                                            of a stream.
+
+                                            This setting cannot be set to `"role:internet"` and `"role:owners"`
+                                            system groups.
+
+                                            **Changes**: New in Zulip 10.0 (feature fevel 319). Previously, the
+                                            resolve topics permission was managed by `can_move_messages_between_topics_group`.
```

Comment:
I think this phrasing would be better written as "Previously, permission to resolve topics was controlled by the more general `can_move_messages_between_topics_group` permission for moving messages."